WordPress/Classes/ViewRelated/NUX/LoginPrologueSignupMethodViewController.swift

```diff
@@ -0,0 +1,57 @@
+class LoginPrologueSignupMethodViewController: UIViewController {
+    /// Buttons at bottom of screen
+    private var buttonViewController: NUXButtonViewController?
+
+    /// Gesture recognizer for taps on the dialog if no buttons are present
+    fileprivate var dismissGestureRecognizer: UITapGestureRecognizer?
+
+    open var emailTapped: (() -> Void)?
+
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        super.prepare(for: segue, sender: sender)
+
+        if let vc = segue.destination as? NUXButtonViewController {
+            buttonViewController = vc
+        }
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureButtonVC()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.navigationController?.setNavigationBarHidden(true, animated: false)
+    }
+
+    private func configureButtonVC() {
+        guard let buttonViewController = buttonViewController else {
```

Comment:
Looking at the design, there are a couple of differences:
1. The designed buttons have a blue text color.
2. The designed buttons have email and google icons.

<img width="250" alt="screen shot 2018-02-15 at 11 43 11 am" src="https://user-images.githubusercontent.com/1816888/36274961-a810979a-1246-11e8-8fcf-bc3ab73fa52b.png">

Reply:
Color: I thought I did it matching the other, but anyway consistency > blueprint. So let's go for consistency, we can review later all of them.

Icon: I think it's mandatory for Google Login ([ref](https://developers.google.com/identity/branding-guidelines)).

My take: if this is ready, ship it and open a separate one to do these changes.